src/renderer/store/handlers/users.ts

```diff
@@ -151,6 +153,9 @@ export const createOrUpdateUser = (payload: {
   const address = identitySelector.address(getState())
   const privKey = identitySelector.signerPrivKey(getState())
 
+  await dispatch(certificatesHandlers.epics.createOwnCertificate(nickname))
```

Comment:
after you refactor this to saga, the code here should be just:
```
dispatch(certificatesActions.creactOwnCertificate(nickname))
```

so in certificates master saga you should have: 
```
takeEvery(certificatesActions.creactOwnCertificate.type, creactOwnCertificate)
```
`creactOwnCertificate` here is a saga that call allt the crypto functions and finnaly does:

```
yield *put(certificatesActions.saveCertificate(ownCertificate))
```